mage_integrations/mage_integrations/destinations/clickhouse/__init__.py

```diff
@@ -1,22 +1,42 @@
 import argparse
-import sys
-
+import traceback
+from sqlalchemy import create_engine
+from clickhouse_sqlalchemy import make_session
 from mage_integrations.destinations.base import Destination
 from mage_integrations.destinations.clickhouse.target_clickhouse.target import (
     TargetClickhouse,
 )
 
 
 class Clickhouse(Destination):
-    def process(self, input_buffer) -> None:
+    def process(self) -> None:
```

Comment:
Let's not change the method signature. https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/mage_integrations/destinations/base.py#L295

Reply:
Modified on 947b0a65dc6207834bed85c022bd8d8efd608b13